snap/snapcraft.yaml

```diff
@@ -27,6 +27,8 @@ environment:
   XDG_DATA_DIRS: $SNAP/usr/share:$SNAP/usr/local/share
   # XKB config from snap
   XKB_CONFIG_ROOT: $SNAP/usr/share/X11/xkb
+  FONTCONFIG_PATH: $SNAP/etc/fonts
+  FONTCONFIG_FILE: $SNAP/etc/fonts/fonts.conf
```

Comment:
From the description this should be `usr/share/fontconfig`. One of them needs fixing

Reply:
Fixed description.